domain-management/src/main/java/org/jboss/as/domain/management/security/adduser/JavaConsole.java

```diff
@@ -24,10 +24,14 @@
 
 package org.jboss.as.domain.management.security.adduser;
 
+import org.aesh.readline.Prompt;
```

Comment:
These imports are in the wrong location, they should be after the static imports but before the java.* imports.